Cargo.toml

```diff
@@ -76,21 +80,25 @@ rustc_version = "0.2"
 [dev-dependencies]
 anyhow = "1.0.19"
 wasmer-wast = { path = "tests/wast" }
+lazy_static = "1.4.0"
```

Comment:
If we avoid patch versions when it doesn't matter to us, we can reduce build times by avoiding duplicated dependencies